edx_when/models.py

```diff
@@ -84,7 +99,7 @@ def actual_date(self):
         if self.abs_date:
             return self.abs_date
         else:
-            return self.content_date.policy.abs_date + timedelta(days=self.rel_date or 0)
+            return self.content_date.policy.actual_date + self.rel_date
```

Comment:
The behavior is still undefined if the original content_date is relative. 
Should edx-when have an opinion about handling relative dates, or is that left to studio to store something sensible?

Reply:
No, this changes the behavior to make it so that `actual_date` returns a `timedelta` if the policy has a relative date, or a `datetime` if the policy has an absolute date. However, it's left up to the caller of what to do if `actual_date` is relative (the external system has to know how to find the content availability date for the user).